samcli/commands/init/interactive_init_flow.py

```diff
@@ -0,0 +1,58 @@
+"""
+Isolates interactive init prompt flow. Expected to call generator logic at end of flow.
+"""
+import click
+
+from samcli.local.common.runtime_template import RUNTIMES, RUNTIME_TO_DEPENDENCY_MANAGERS
+from samcli.commands.init.init_generator import do_generate
+from samcli.commands.init.init_templates import InitTemplates
+
+
+def do_interactive(location, runtime, dependency_manager, output_dir, name, app_template, no_input):
+    if location:
```

Comment:
Can we convert if elses into: dictionary dispatch tables? https://medium.com/better-programming/dispatch-tables-in-python-d37bcc443b0b